components/net_traits/lib.rs

```diff
@@ -476,6 +477,7 @@ pub enum ResourceAttribute {
     FetchStart,
     ConnectStart(u64),
     ConnectEnd(u64),
+    SecureConnectionStart(u64),
```

Comment:
No need to pass a value here - like many other values, we can default to precise_time_ns().